<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="resourceDiff">
         <property name="text">
          <string>resources_diff</string>
         </property>
        </widget>
        <widget class="QPushButton" name="cubePoly">
         <property name="text">
          <string>cubePoly</string>
         </property>
        </widget>
        <widget class="QPushButton" name="tracePoints">
         <property name="text">
          <string>tracePoints</string>
         </property>
        </widget>
        <widget class="QPushButton" name="decode">
         <property name="text">
          <string>decode</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="cityRoadMap">
         <property name="text">
          <string>cityRoadMap</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bezierPoly">
         <property name="text">
          <string>bezierPoly</string>
         </property>
        </widget>
        <widget class="QPushButton" name="cityPosition">
         <property name="text">
          <string>cityPosition</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="xiaoqianCollect">
         <property name="text">
          <string>xiaoqianCollect</string>
         </property>
        </widget>
        <widget class="QPushButton" name="convertDialogue">
         <property name="text">
          <string>gcld convertDialogue</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="utf8Bom">
         <property name="text">
          <string>utf-8 BOM to without BOM</string>
         </property>
        </widget>
        <widget class="QPushButton" name="rc4">
         <property name="text">
          <string>rc4</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tqcfConvertDialog">
        <property name="text">
         <string>tqcf convertDialog</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
